chrome_extension/README.md

```diff
@@ -0,0 +1,44 @@
+# Oak Chrome Extension
+
+The Oak Chrome Extension is a proof of concept to show how Oak may be used as
+part of web applications.
+
+A regular web page may at any point enter Oak private mode, which means that
+from that point onwards it will be able to communicate exclusively with an Oak
+application that can enforce that any data it receives is kept private for the
+current session. This is similar to "incognito mode" but for the server-side
+parts of an application.
+
+A web page enters Oak private mode by invoking the following Javascript code:
+
+```js
+window.oak.enter();
```

Comment:
Note that this conceivably overlaps with websites that use this key already. Are there other planned properties for the global `oak` object? Would it make sense to use `window.postMessage("oak_enter")` instead?

Reply:
Fair enough, I was trying to make it simpler for web pages, but I think sticking to `postMessage` actually also avoids the need of extra weird injected scripts, so let's go with that.